app.config.ts

```diff
@@ -61,7 +61,10 @@ export default ({ config }: ConfigContext) =>
       plugins: [
         ["./scripts/fdroid/removeDKBuildId.ts"],
         ["./scripts/fdroid/disableDependencyMetadata.ts"],
-        ["./scripts/fdroid/excludeNonfreeDependencies.ts"],
+        [
+          "./scripts/detox/configureDetox.ts",
+          // { subdomains: "*" }, // uncomment to debug app
```

Comment:
```suggestion
          { subdomains: "*" }, // uncomment to debug app
```

@naltatis I think you have to uncomment this line to build and test the app on Android.
For this PR we can temporarily do that and find a nicer solution if the actions succeed.